Cogwheel Skull

Nik-Was opened the pull request:
Adds the coghweel skull as a placeable block and potato cannon projectile (Applies a "time stop" AOE on impact) 
Also fixes some block name inconsistencies

Nik-Was committed:
Added craftable and wrenchable Cogwheel Skull block

Nik-Was committed:
Merge branch 'Laskyyy:Astral-Experimental' into Astral-Experimental

Nik-Was committed:
Added Cogwheel Skull as a Potato Cannon projectile with a "time stop" effect

nageih commented:
Please add the name of the item into language file, referring to the [doc](https://github.com/Laskyyy/Create-Astral/blob/Astral-Experimental/CONTRIBUTING.md#internationalisation-and-localisation-support)

The-Shortman commented:
Marking this as a draft so I don't accidentally merge it, please mark it as ready for review once the lang keys have been added

Nik-Was committed:
Merge branch 'Laskyyy:Astral-Experimental' into Astral-Experimental

Nik-Was committed:
Made cogwheel skull also breakable with axe, like regular cogwheels
Removed redundant line in the ammo definition
Added a tooltip along with the name to lang file

Additionally unified all names for "Blast-Resistant" and "Fire-Resistant" blocks into one format

Nik-Was committed:
Merge remote-tracking branch 'origin/Astral-Experimental' into Astral-Experimental

nageih commented on kubejs/startup_scripts/tooltip/createastral.js:
```diff
@@ -41,6 +41,7 @@
       },
     ];
     createastralTooltips.forEach((tooltip) => addLocalizedShiftTooltip(event, tooltip.item, tooltip.tooltip));
+    event.add("createastral:cogwheel_skull", Text.of(Component.translate("tooltip.createastral.cogwheel_skull")).gray());
```
Oh, I just saw this code. I think I saw some issues. Maybe we need to do some testing?
Including the following, I think adding `Component.translate` to `Text.of` is strange.
I think the normal approach should be `Component.translate("tooltip.createastral.cogwheel_skull").gray()`

nageih replied:
Gosh, this was what I did a year ago. What an idiot!

Nik-Was replied:
Yeah I don't actually know how any of that works, just copied from the code below. Can remove it if you think it shouldn't be there, but I don't think it affects the end result

The-Shortman reviewed (approved):
Everything looks good to me

The-Shortman commented:
I think I can safely merge this pull request
